llpc/context/llpcCompiler.cpp

```diff
@@ -268,8 +268,9 @@ class LlpcDiagnosticHandler : public DiagnosticHandler {
 // @param optionCount : Count of compilation-option strings
 // @param options : An array of compilation-option strings
 // @param [out] ppCompiler : Pointer to the created LLPC compiler object
+// @param pCache : Pointer to ICache implemented in client
 Result VKAPI_CALL ICompiler::Create(GfxIpVersion gfxIp, unsigned optionCount, const char *const *options,
-                                    ICompiler **ppCompiler) {
+                                    ICompiler **ppCompiler, Vkgc::ICache *pCache) {
```

Comment:
I don't think you need Vkgc:: prefix in this cpp file because we have already specified "using namespace Vkgc;". Please remove it in all codes of this file.